src/ripple/net/impl/RPCCall.cpp

```diff
@@ -1019,7 +1019,21 @@ int RPCCall::fromCommandLine (const std::vector<std::string>& vCmd)
         }
         else
         {
-            auto const setup = setup_ServerHandler(getConfig(), std::cerr);
+            ServerHandler::Setup setup;
+            try
+            {
+                std::stringstream ss;
+                setup = setup_ServerHandler(getConfig(), ss);
+            }
+            catch(...)
+            {
+                // eat this
```

Comment:
Make no mistake, this is a hack to fix rpc-port. We are going to move the command line client out of rippled soon anyway.